.github/workflows/nightly-builds.yml

```diff
@@ -145,25 +145,18 @@ jobs:
         # binary version is required and Akka build will set the right
         # full version from it.
         scalaVersion: ["2.12", "2.13"]
-        jdkVersion: ["adopt@1.8.0", "adopt@1.11", "openjdk@1.17.0"]
-        include:
-          - jdkVersion: adopt@1.8.0
-            extraOpts: ""
-          - jdkVersion: adopt@1.11
-            extraOpts: ""
-          - jdkVersion: openjdk@1.17.0
-            extraopts: ""
+        javaVersion: [8, 11, 17]
```

Comment:
Not sure about the numeric part, where did you get that from? E.g. see https://github.com/actions/setup-java#testing-against-different-java-versions

Reply:
numeric values work (as do text ones) - https://github.com/apache/incubator-pekko-connectors-kafka/pull/26 is an example where I use numeric values (in a previous PR in pekko space)